infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -183,6 +212,100 @@ public static void main(String[] args) {
         addManagementServer();
         router.baseServer.setOptionsMap(opts);
 
+        // Setup SSL if needed
+        Boolean tlsEnabled = (Boolean) opts.get("--enable-tls");
+        if (tlsEnabled) {
+            // Get the key store password
```

Comment:
Thanks for looking at this PR Scott.
+1 for the refactoring.
Here are the key stores and trust stores I used for manual test:
https://github.com/CorfuDB/CorfuDB/compare/76f9ed5...CorfuDB:tls-support